internal/namespaces/k8s/v1beta4/testdata/test-get-version-simple.stdout.golden

```diff
@@ -0,0 +1,22 @@
+region                          fr-par
+name                            1.17.4
```

Comment:
shouldn't name be the first field ?

Reply:
Hmm yep, no clue why region is here though :thinking:

Reply:
Because of the SDK 

https://github.com/scaleway/scaleway-sdk-go/blob/master/api/k8s/v1beta4/k8s_sdk.go#L867 

Should we reorder the fields in the SDK first ?